<!-- -->
<launch>
	<node pkg="camlasercalib" type="lidar_cam" name="cam_lidar_calib" args="$(find camlasercalib)/../transfm/imageCloudPoints.txt $(find camlasercalib)/../transfm/param/calib.yml" output="screen">
		    <param name="strSub_pc2" type="string" value="/velodyne32/velodyne_points"/>
			<param name="strSub_img" type="string" value="/camera/image_color"/>
			<param name="onlyDrawPointsColor" type="bool" value="false"/>
		    <param name="DistanceThreshold" type="double" value="0.05"/>
	</node>
</launch>
